examples/feathersjs/README.md

```diff
@@ -0,0 +1,36 @@
+# Example app with [feathersjs](https://feathersjs.com/)
+
+Feathersjs is a framework for real-time node applications and this example will show you how to connect to the feathers backend from remix and use it to view events, borrowing heavily from the socket.io example and adapting it for feathersjs. 
+
+To use run `npm run watch` to build and watch for remix changes and `npm run start` to start our server
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+<!-- TODO: update this link to the path for your example: -->
```

Comment:
The codesandbox link must be updated to match your example path

Reply:
done